1.	Identify-Basic-Internet-Problems-with-ping.md

```diff
@@ -0,0 +1,113 @@
+<img src="https://media.licdn.com/mpr/mpr/jc/AAEAAQAAAAAAAAfoAAAAJDRmY2I4ZmE1LTk4ZmQtNGUzOS1hOWQ4LTNkMGJkOWQxNjNkYg.png" width="400"  alt="Sonar screen">
+
+## Identify Basic Internet Problems with "ping"
+
+Next time you call your help desk, do you want to wow them with your networking knowledge? Using a command called “ping”, built right into your existing Mac or Windows computer, will help identify basic connection problems. Okay, this might not be enough to “wow” your fellow team members, however they will appreciate that you started the debug process.
+
+**The "ping" command:**
```

Comment:
Make these as headings.  
If you want to show them as small headings, then use heading 3, 4 or 5.  
i.e.
```md
##### The "ping" command:
```
which would show as 
> ##### The "ping" command: